.github/workflows/build-and-test.yml

```diff
@@ -43,9 +43,9 @@ jobs:
           use-mamba: true
       - name: Create installer
         run: |
-          CONDA_SUBDIR=${{ matrix.target_arch }} mamba create --name constructor-${{ matrix.target_arch }} conda-standalone
-          CONDA_STANDALONE_PATH="$(conda info --envs | grep constructor-${{ matrix.target_arch }} | sed -E 's@([^ ]+ +)@@g')/standalone_conda/conda.exe"
-          pip install git+http://github.com/chrisburr/constructor.git@diracos2-patches
+          CONDA_SUBDIR=${{ matrix.target_arch }} mamba create -c chrisburr/label/micromamba-dev --name constructor-${{ matrix.target_arch }} micromamba
```

Comment:
That's not exactly something for production.

Reply:
Oops I reverted that but didn't force push, fixed in https://github.com/DIRACGrid/DIRACOS2/pull/67